tests/datatypes/test_bytes.py

```diff
@@ -28,6 +30,12 @@ def test_capitalize(self):
             print(b'2015638687'.capitalize())
         """)
 
+    @unittest.skip("Move this to test_capitalize upon resolution of #530")
```

Comment:
Sorry - you're right - I got `skip` and `skipIf` confused. The point is the same though - skip will *never* run the test; expectedFailure will run it, but accept it when it fails. When, at some point in the future, the bug is fixed, the test will start passing - and the expectedFailure will cause an "unexpected success" - which means someone will be forced to look at that line of code (and the comment), and correct the problem, merging the test cases.